File: ms-auth-java/domain/model/src/main/java/co/com/bancolombia/model/signin/SignIn.java
package co.com.bancolombia.model.signin;
import co.com.bancolombia.model.session.Session;
import co.com.bancolombia.model.user.User;
import lombok.Builder;
import lombok.AllArgsConstructor;
import lombok.Getter;
//import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
//@NoArgsConstructor
@AllArgsConstructor
@Builder(toBuilder = true)
public class SignIn {
    private User user;
    private Session session;
}

File: ms-auth-elixir/lib/domain/model/bancolombia/signin/sign_in.ex
defmodule Authelixir.Domain.Model.Bancolombia.Signin.Signin do
  @moduledoc "Modelo de entrada para SignIn. Envoltorio de User y Session."

  alias Authelixir.Domain.Model.Bancolombia.User.User
  alias Authelixir.Domain.Model.Bancolombia.Session.Session

  defstruct [:user, :session]

  @type t :: %__MODULE__{
          user: User.t(),
          session: Session.t() | nil
        }
end
